django_loci/base/models.py

```diff
@@ -85,7 +85,7 @@ def short_type(self):
 
 class AbstractFloorPlan(TimeStampedEditableModel):
     location = models.ForeignKey('django_loci.Location', on_delete=models.CASCADE)
-    floor = models.SmallIntegerField(_('floor'))
+    floor = models.SmallIntegerField(_('floor'), null=True)
```

Comment:
Well, it works alright, but we should add a condition that it can't be null. (`null-false, blank=false`), because a floor always has number, right?

What do you think?

Reply:
Yes I figured. Should I add a default value as well?